### ISSUE TITLE
Tagging without multiple with new tags doesn't works when using angularjs 1.3.2

### ISSUE THREAD

**Issue by larrybolt** [angular-ui/ui-select#890]
When using angularjs 1.3.2 tagging without multiple doesn't work for new items that are not added to the array.
I've also tried editing `demo-tagging.html`, by replacing the angularjs libraries with following:
```
<script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.3.2/angular.js"></script>
<script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.3.2/angular-sanitize.js"></script>
```
And it seems to confirm something changed in the 1.3.2 to break this feature.

I'm really hoping there's an easy fix for this :disappointed: as this feature was exactly what I needed.

**Comment by larrybolt** [comment]
And just trying to use angularjs 1.3.0 on the app I'm working on does run the tagging function, but doesn't seem to pass any arguments such as the new object.
``` Javascript
$scope.tagTransform = function (newTag) {
    console.log("new tag", newTag, arguments);
    // ...
  };
```

**Comment by grebett** [comment]
+1

**Comment by grebett** [comment]
I've written a quick workaround you can find [there](http://pastebin.com/djfdj4R1).
Not sure it's good enough for a pull request but may do the job for the while.

**Comment by emreavsar** [comment]
+1 same issue here.

**Comment by nutanabhiram** [comment]
I am facing the same issue , tagging is not enabling with following code using angularjs 1.3.2 
<ui-select tagging tagging-label="('new')" ng-model="singleDemo.color" theme="bootstrap" title="Choose a color">
    <ui-select-match placeholder="Select color...">{{$select.selected}}</ui-select-match>
    <ui-select-choices repeat="color in availableColors | filter: $select.search">
      {{color}}
    </ui-select-choices>
</ui-select>

**Comment by karptonite** [comment]
I actually see this same problem in 1.3.1 as well. Works fine in 1.3.0.

**Comment by guilhermerocha** [comment]
I'm facing the same problem.

**Comment by ThomasAlber** [comment]
same problem here
you can reproduce the problem in the official example

http://plnkr.co/edit/m1SQXUxftBLQtitng1f0?p=preview

by changing the angular version in line 21 from
```<script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.3.0/angular.js"></script>```
to
```<script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.3.1/angular.js"></script>```
now tagging of a single-select isn't working anymore
i also tried
```<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.0-rc.2/angular.min.js"></script>```
and it's not working

**Comment by Sitaxa** [comment]
Same problem  here. Any issue ?

**Comment by sukantary** [comment]
I'm facing the same problem. Please help

**Comment by karptonite** [comment]
This temporary fix worked for me:
https://github.com/angular-ui/ui-select/issues/983#issuecomment-109948338

**Comment by LeleDev** [comment]
Please make tagging work also without the workaround tagging-label="false"...

**Comment by filippolorenzoferraris** [comment]
:+1:

**Comment by davekr** [comment]
I solved this with older version of ui-select. I am using version 0.11.0. With angular >1.3.2 it works as it's suppoused to and with angular <=1.3.2 you have to set tagging-label to "(new)" explicitly otherwise "undefined" will be shown.
